src/resources/js/commerce/gateway/paypal/checkout.js

```diff
@@ -358,10 +358,6 @@ tribe.tickets.commerce.gateway.paypal.checkout = {};
 	 * @return {void}
 	 */
 	obj.setupLoader = function() {
-
-		// Show loader.
-		$document.trigger( tribe.tickets.commerce.customEvents.showLoader );
```

Comment:
If we keep the loader template as it was, we can do a `show` here. That's how we do it for the other places where we make use of the loader.

Reply:
Yes, that is what I thought until I found that we are not getting into the `ready` state while the `PayPal Checkout` script is getting loaded. So, it keeps showing blank and by the time we trigger the loader to show the buttons are already rendered and the loader doesn't help at all.

So, I have modified the loader to be visible at page render immediately while the JS scripts get loaded.